src/core/server/base.js

```diff
@@ -22,7 +22,7 @@ import { createApiError } from 'core/api';
 import ServerHtml from 'core/containers/ServerHtml';
 import { prefixMiddleWare, trailingSlashesMiddleware } from 'core/middleware';
 import { convertBoolean } from 'core/utils';
-import { setClientApp, setLang, setJwt, setUserAgent } from 'core/actions';
+import { setClientApp, setLang, setAuthToken, setUserAgent } from 'core/actions';
```

Comment:
Nitpick but we tend to alphabetise imports. But like, only fix if you need to fix something else too.